tests/models/hf/test_hf_config.py

```diff
@@ -83,7 +83,8 @@ def test_tie_weights(tie_word_embeddings: bool):
     },
     {
         'attn_config': {
-            'attn_impl': 'triton'
+            'attn_impl': 'flash',
+            'alibi': False,
```

Comment:
why did you need to add alibi false?

Reply:
it looks like flash2 isn't installed for gpu tests?

Reply:
And alibi isn't supported with flash2

Reply:
flash2 is installed for gpu tests

Reply:
and should work fine with alibi

Reply:
ah whoops this was for cpu tests which don't have flash2 installed

Reply:
fixed with a patch